homeassistant/components/hive/sensor.py

```diff
@@ -13,7 +14,32 @@
 PARALLEL_UPDATES = 0
 SCAN_INTERVAL = timedelta(seconds=15)
 DEVICETYPE = {
+    "Camera_Temp": {
```

Comment:
I suggest refactoring to use entity descriptions instead. See eg:
https://github.com/home-assistant/core/blob/ac11a9b7ff3c8b62e648d351a2b63125f71fad1d/homeassistant/components/airly/sensor.py#L109-L116

Reply:
I’ll take a look at this

Reply:
I have updated this now to use the example provided